taichi/ir/ir.h

```diff
@@ -824,6 +824,10 @@ class GlobalPtrStmt : public Stmt {
     return activate;
   }
 
+  bool common_statement_eliminable() const override {
```

Comment:
Just to confirm: if there's activation, CFG optimization will only eliminate the second GlobalPtr, right? (assuming the first statement always precedes the second statement...)

Reply:
Yes.